src/HttpServer/Response.gren

```diff
@@ -73,12 +81,26 @@ setHeader key value (Response responseKey response) =
         }
 
 
-{-| Set the body on a response.
+{-| Alias for [`setBodyToString`](HttpServer#setBodyToString).
 -}
 setBody : String -> Response -> Response
-setBody body (Response key response) =
+setBody = setBodyToString
+
+
+{-| Set the body of the response to a string.
+-}
+setBodyToString : String -> Response -> Response
+setBodyToString body (Response key response) =
     Response key
-        { response | body = body }
+        { response | body = StringBody body }
+
+
+{-| Set the body of the response to some bytes.
+-}
+setBodyToBytes : Bytes -> Response -> Response
+setBodyToBytes body (Response key response) =
```

Comment:
Bear in mind that English is not my native language, but would `setBodyAsBytes` be better? 🤔

Reply:
I think so, yes.